packages/pn-personafisica-webapp/src/components/Contacts/DigitalContactElem.tsx

```diff
@@ -253,6 +272,7 @@ const DigitalContactElem = forwardRef<{ editContact: () => void }, Props>(
                   type="button"
                   onClick={editHandler}
                   sx={{ mr: 2 }}
+                  id="saveModifyButton"
```

Comment:
Be aware this id is not unique. You can use the concatenation between this string and f.id

Reply:
use the same concatenation  as for modifyContact and cancelContact

Reply:
done